### DISCUSSION TITLE
Episode 177: The Couch Episode

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#69] [Episode]
Episode Link: https://adspthepodcast.com/2024/04/12/Episode-177.html

Feel free to write comments or ask questions 🙂

**Comment by wilhelm-now** [comment]
I usually listen at 1.6x, dialed it down to 1.25 to make sure I didn't miss anything.

**Comment by BraxtonMDevelops** [comment]
This was amazingly well timed am going couch hunting with my mom on Sunday. So neat to hear about bryces opinions on couches. Cute story about Bryces dog learning to manipulate their humans.

**Comment by pascal754** [comment]
Cozy couches + nice views inspire ADSP.

**Comment by tothambrus11** [comment]
Great episode, I love it :D
I would have never thought a sane person would ever give out 5K for a single piece of furniture. This is one of the most convincing ways of promoting NVidia as an employer.

I'm a student in the Netherlands, and we got almost all of our furniture for free from a second-hand community website, including two almost perfect couches. I believe the wife didn't like the color or something...

**Comment by CharString** [comment]
After the live coding sessions and the excellent Tacit series, this is way way up there in the "what are they talking about" axis... It took a while before I understood what that chez lounge was, that Bryce was talking about. I only knew the thing as a chaise longue, in European French pronunciation. https://www.youtube.com/watch?v=nHJuy7Ws3m8 Top notch content... I found out Spotify only goes to 3.5x but well worth doing it on the second listen through.

**Comment by VincentZalzal** [comment]
I just want Bryce to know that, when I decided that this episode wasn't for me, I didn't skip the episode : I 10x'ed it ;)
